app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
>Thirdly, if we're going to choose one approach to be able to "extend" to cover all cases, from a maintainability point of view I would strongly argue in favour of basing it on top of a proven, object-oriented, heavily tested & highly documented forms implementation like wtforms rather than a home-baked solution which is not really object-oriented and already a little hard for a newbie to decode. I'm confident wtforms would be able to build its fields from the content loader if necessary.

I think it's important to understand that you're making a choice in the context of an existing project. This is not the first form we're building and it's not the first filter we're building, so the question is not "what technology would I personally prefer" it's "what makes sense for this project and this team".

And I think that the important part is that we have dozens of forms driven by content-loader and frontend-toolkit, including one working implementation of filters. It is proven, it is tested, and the team as a whole has more experience with it than with wtforms.

Setting that aside, I'd rather we have one approach to filter implementation than two. It feels like adding lot and      status questions to the frameworks is not really a blocker, since we've done this for G-Cloud before. And the argument for extending it to filter on other brief fields is less abstract than it is with wtforms – we have a working example.

I'd love to hear what everyone else thinks (esp @tombye as he knows the G-Cloud filters much better than I do), but my position is: unless there's a lot more work involved in going with the content-loader route (or if we think it'd be a good idea to use wtforms for G-Cloud filters) I think we should make two implementations as similar as possible.